src/Illuminate/Mail/Events/MessageSent.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Illuminate\Mail\Events;
+
+use Swift_Message;
+
+class MessageSent
+{
+    /**
+     * The Swift message instance.
+     *
+     * @var \Swift_Message
+     */
+    public $message;
+
+    /**
+     * The Swift transport.
+     *
+     * @var string
+     */
+    public $transport;
+
+    /**
+     * The Swift send result - format depends on the transport.
```

Comment:
The issue I have with this is that our mail system is not an abstraction if we allow this.